osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/config/ThrottlingClientConfig.java

```diff
@@ -48,22 +50,19 @@ public String configurationName() {
   @Bean(destroyMethod = "unregister")
   @Conditional(ThrottlingClientEnabledCondition.class)
   public ThrottlingClient throttlingClient() {
-    final ThrottlingClient throttlingClient =
-        new ThrottlingClient(
-            new org.opensmartgridplatform.throttling.api.ThrottlingConfig(
-                this.configurationName, this.configurationMaxConcurrency),
-            this.throttlingServiceUrl,
-            this.timeout);
-    throttlingClient.register();
-    return throttlingClient;
+    return new ThrottlingClient(
+        new org.opensmartgridplatform.throttling.api.ThrottlingConfig(
```

Comment:
I just noticed the fully qualified reference to the `ThrottlingConfig` constructor is probably a leftover from earlier versions of the code were `ThrottlingClientConfig` was called `ThrottlingConfig` as well.
If you do something about the other comment you could consider removing the package name and importing the `ThrottlingConfig` type.

Reply:
changed